docs/installation/debian.md

```diff
@@ -75,6 +75,11 @@ from the new repository:
 
             deb https://apt.dockerproject.org/repo debian-stretch main
 
+> **Note**: Docker does not provide packages for all architectures. To install docker on
+> a multi-architecture system, add an `[arch=...]` clause to the entry. Refer to the
+> [Debian Multiarch wiki](https://wiki.debian.org/Multiarch/HOWTO#Setting_up_apt_sources)
+> for details.
+
```

Comment:
Can you indent this block with 4 spaces, otherwise the numbered list is split, and step 8. becomes 1.

note; you can check the rendered documentation by running `make docs` in the root of your repo, it will build the docs in a container and start a webserver to check the output, pretty neat